### ISSUE TITLE
Is there a way to know if test case is in rerun state?

### ISSUE THREAD

**Issue by chilakamarthi** [microsoft/vstest#3659]
When I run a test case, if it fails in pipeline and I set rerun to 3, is there a way to know in subsequent tries that the execution is rerun?

**Comment by nohwnd** [comment]
I don't think you can know it, the re-run in pipeline takes the same test and runs it on vstest, but vstest does not know that this is a re-run.
